packages/blade/src/components/Tooltip/types.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type { Side, UseFloatingOptions } from '@floating-ui/react';
+import type { CSSProperties } from 'react';
+import type { Platform } from '~utils';
+import type { BaseBoxProps } from '~components/Box/BaseBox';
+
+type TooltipProps = {
+  content: string;
+  placement?: Exclude<
+    UseFloatingOptions['placement'],
+    'left-end' | 'left-start' | 'right-end' | 'right-start'
+  >;
+  children: React.ReactElement;
+  onOpenChange?: ({ isOpen }: { isOpen: boolean }) => void;
+};
+
+type TooltipTriggerProps = {
+  onBlur?: Platform.Select<{
+    native: undefined | ((event: any) => void);
+    web: React.FocusEventHandler;
+  }>;
+  onFocus?: Platform.Select<{
+    native: undefined | ((event: any) => void);
+    web: React.FocusEventHandler;
+  }>;
+  onMouseLeave?: Platform.Select<{ web: React.MouseEventHandler; native: undefined }>;
+  onMouseMove?: Platform.Select<{ web: React.MouseEventHandler; native: undefined }>;
+  onPointerDown?: Platform.Select<{ web: React.PointerEventHandler; native: undefined }>;
+  onPointerEnter?: Platform.Select<{ web: React.PointerEventHandler; native: undefined }>;
+  onTouchStart?: Platform.Select<{
+    native: undefined | ((event: any) => void);
+    web: React.TouchEventHandler;
+  }>;
+  onTouchEnd?: Platform.Select<{
+    native: undefined | ((event: any) => void);
+    web: React.TouchEventHandler;
+  }>;
+};
+
+type TooltipContentProps = {
+  children: React.ReactNode;
+  style: CSSProperties;
+  arrow: React.ReactNode;
```

Comment:
what is this arrow for?

Reply:
I pass the Tooltip arrow component to render inside of this TooltipContent, 

```
<TooltipContent
  style={styles}
  arrow={
    <FloatingArrow
      ref={arrowRef}
      context={context}
      ...
    />
  }
>
  {content}
</TooltipContent>
```